app/db/sqlite.py

```diff
@@ -0,0 +1,21 @@
+from app import sqlite_db as db
+
+
+# This class will automatically create a table in the SQLite database if it doesn't exist, insert a row into that table
+# when a new instance of it gets made with new values, or edit an existing row under the right conditions.
+class Users(db.Model):
```

Comment:
sqlite databases shouldn't really exist in the first place. They technically should live within our Oracle/MySQL databases. There are some exceptions, but that should be the standard.